### ISSUE TITLE
AMD GPU Problems 

### ISSUE THREAD

**Issue by Nini002** [RPCS3/rpcs3#13224]
### Quick summary

AMD GPU Driver problems 

### Details

first off I still have this error 
E RSX: Your GPU does not support alpha-to-one for multisampling. Graphics may be inaccurate when MSAA is enabled

Secondly
Valkan is broken on AMD  both of my  GPUS are affected an Rx 5500xt 8gig and RX 5600xt,
when using Vulkan Games boot when and if they want, and when they do boot if I'm using Asynchronous texture streaming my pc will crash to the point where I must flip the power switch on the PSU to shut it down. This has happened on all the GPU drivers and all the RPCS3 builds of 2022 

### Attach a log file

[RPCS3.zip](https://github.com/RPCS3/rpcs3/files/10415630/RPCS3.zip)

### Attach capture files for visual issues

![unknown_2023 01 13-17 40](https://user-images.githubusercontent.com/72757314/212424660-cce673e6-ab54-4a7c-ba6e-fff9271d6285.png)

### System configuration

PC Specs 
MOBO - Pro b660m-a ddr4
OS- win 11 pro Version	10.0.22621 Build 22621
GPU- RX5600xt driver version 22.11.2
CPU- i5  12400f 
Ram- 16 gigs@ 3000MHz
RPCS3 build -0.026-14568-1852b370 Alpha|master

**Comment by AniLeo** [comment]
alpha-to-one missing on AMD windows driver is expected and if asynchronous texture streaming doesn't work then disable it. If you have GPU crashes then this is not an RPCS3 issue as userspace applications must not be able to crash a driver.

**Comment by kd-11** [comment]
1. Do not use the "Fast" asynchronous scheduler unless you know what you're doing. It's in the advanced tab for a reason. I disabled it on NVIDIA for the same reason, it will hang your GPU (and on NVIDIA causes BSOD)
2. The error in red is simply a hardware check, no games actually rely on that feature that we've found.

Run the emulator with the default recommended settings, adding accuracy options as needed. The performance options are there for those that can take advantage of them, but if you encounter problems, that's expected.
